homeassistant/components/homeassistant/__init__.py

```diff
@@ -116,6 +121,12 @@ async def async_handle_turn_service(service):
 
     service_schema = vol.Schema({ATTR_ENTITY_ID: cv.entity_ids}, extra=vol.ALLOW_EXTRA)
 
+    hass.services.async_register(
+        ha.DOMAIN,
+        SERVICE_SAVE_PERSISTENT_STATES,
+        async_save_persistent_states,
+        schema=service_schema,
```

Comment:
The new service doesn't take any data parameters so we don't need to specify a validation schema.

Reply:
Removed.
A side note: Should I be ammending my previous commit or adding a new commit when make changes to a PR ?

Reply:
Please just add new commits after review has started. Amending or squashing makes it harder for readers to track changes. We'll squash when merging. Thanks.